backend-project/requirements/development.txt

```diff
@@ -1,3 +1,4 @@
 -r production.txt
 django-debug-toolbar==2.1
 factory-boy==2.12.0
+lxml==4.4.2
```

Comment:
Dlaczego?

Reply:
Zeby sobie lokalnie pobrac TERC i to drugie.